src/devices/CharacterLcd/LcdCharacterEncodingFactory.cs

```diff
@@ -0,0 +1,929 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace Iot.Device.CharacterLcd
+{
+    /// <summary>
+    /// Factory for creating Encodings that support different cultures on different LCD Displays. 
+    /// </summary>
+    public class LcdCharacterEncodingFactory
+    {
+        private static Dictionary<char, byte> DefaultA00Map;
+        private static Dictionary<char, byte> DefaultA02Map;
```

Comment:
static fields should be called like: `s_defaultA02Map`

Reply:
They should be static readonly, so I think that's right like it is?

Reply:
I think fine to keep a name with static readonly